client/lxd_containers.go

```diff
@@ -809,12 +809,16 @@ func (r *ProtocolLXD) GetContainerFile(containerName string, path string) (io.Re
 	}
 
 	// Parse the headers
-	uid, gid, mode, fileType, _ := shared.ParseLXDFileHeaders(resp.Header)
+	headers, err := shared.ParseLXDFileHeaders(resp.Header)
+	if err != nil {
+		return nil, nil, err
```

Comment:
We should wrap the error here using `return nil, nil, fmt.Errorf("Failed parsing response headers: %w", err)`